src/index.html

```diff
@@ -6,8 +6,45 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+     class="card"
+     data-qa="card"
+    >
+
+      <img class="card__img"
+           src="images/imac.jpeg"
+           alt="APPLE A1419 iMac 27"
+      >
```

Comment:
Take a look at the [checkList](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)
![image](https://github.com/mate-academy/layout_product-cards/assets/91411692/8a806d35-dc67-497f-a314-45e3530edc2e)

Reply:
fix it in all cases